apps/mobile/app/(route)/index.tsx

```diff
@@ -1,17 +1,39 @@
+import { WEBVIEW_MESSAGE_TYPES } from "@repo/constants";
+import { useRef } from "react";
 import { WebView, WebViewMessageEvent } from "react-native-webview";
 
 import { useHandleNavigationActions } from "@/hooks/useHandleNavigationActions";
-import { getBaseUrl } from "@/utils/getBaseUrl";
+import { handleReceiveMessage, handleLoadProgressCurried, handleLoadCurried } from "@/utils/bridgeHandler";
+import { getNativeApiUrl } from "@/utils/getNativeApiUrl";
+import { parseMessage } from "@/utils/parseMessage";
 
 export default function HomeScreen() {
-  // TODO : login상태에 따른 분기 처리 설정
+  const webviewRef = useRef<WebView>(null);
 
-  const baseUrl = getBaseUrl();
+  const baseUrl = getNativeApiUrl();
   const handleNavigationActions = useHandleNavigationActions();
 
-  const requestOnMessage = (e: WebViewMessageEvent) => {
-    handleNavigationActions(e);
+  const handleWebviewMessage = (e: WebViewMessageEvent) => {
+    const { type } = parseMessage(e);
+
+    if (type === WEBVIEW_MESSAGE_TYPES.ROUTER_EVENT) {
+      handleNavigationActions(e);
+    } else {
+      handleReceiveMessage(e);
+    }
```

Comment:
```suggestion
  const handleOnMessage = (e: WebViewMessageEvent) => {
      handleNavigationActions(e);
      handleReceiveMessage(e);
```

어떤 type인지는 함수 안에서 판단할거같으니까 if문을 삭제하고 그냥 작성하는건 어떨까?